cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/store/NoSqlRepositorySourceControlMetadataStoreTest.java

```diff
@@ -33,6 +34,11 @@ public static void beforeClass() throws Exception {
     transactionRunner = injector.getInstance(TransactionRunner.class);
   }
 
+  @Override
+  @Test
+  public void testScanRepositorySourceControlMeta() {
```

Comment:
Please add a comment on why it does not work and why it is empty

Reply:
sure, done